src/Dto/EmsApi.Dto.csproj

```diff
@@ -1,14 +1,15 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
How come this one does not target 2.1 as well?

Reply:
I was having some issues with parallelism when generating and modifying the source file, but I did finally find the magic combination of msbuild things to make it work, so this project targets both now